tests/grammar/test_grammar.py

```diff
@@ -116,7 +117,8 @@ def from_grammar(start: str = "file_input") -> st.SearchStrategy[str]:
 )
 @hypothesis.settings(
     deadline=400,
-    max_examples=5000
+    max_examples=5000,
+    suppress_health_check=(HealthCheck.too_slow, )
```

Comment:
About how long are these tests taking?

Reply:
Well we can select what we want for the CI; but I prefer a very high number of examples whilst testing.

Reply:
For sure. Eventually I want to split out hypothesis into its own CI passes so it goes a bit faster between the two sets of testing.

Reply:
for the time being leaving it at 500 seems reasonable to me?

Reply:
Fair enough. I'm also not sure if Github CI has a persistent cache, meaning hypothesis might not get unique examples on every run

Reply:
Shit, it nearly doubled our CI time!

Reply:
I'd like to separate out Hypothesis tasks from the regular testing tasks before this gets merged

Reply:
We can make it 100 then (I think that is the default)? I think the way to do is using xfail the test; and the specifically select it on the separate action?

Reply:
I am way more confident in the Grammar now; because I know that all our get_contract code passes :) I just needed the "high fuzz" rate when I was trimming the python3 grammar down.